data/random-teams.ts

```diff
@@ -1011,7 +1012,8 @@ export class RandomTeams {
 			return {cull: hasMove['dragondance']};
 		case 'hurricane':
 			// Special case for Noctowl, which wants Air Slash if Nasty Plot instead
-			return {cull: counter.setupType === 'Physical' || (hasAbility['Tinted Lens'] && counter.setupType && !isDoubles)};
+			const noctowlCase = (!isNoDyna && hasAbility['Tinted Lens'] && counter.setupType && !isDoubles);
```

Comment:
```suggestion
			const noctowlCase = (!isNoDyna && !isDoubles && species.id === 'noctowl' && counter.setupType);
```
This rejection seems to be unrelated to the actual Tinted Lens ability.